lib/logger/test/logger/translator_test.exs

```diff
@@ -473,7 +473,8 @@ defmodule Logger.TranslatorTest do
       send(pid, :go)
       receive do: ({:DOWN, ^ref, _, _, _} -> :ok)
     end) =~ ~r"""
-    Ancestors: \[#PID<\d+\.\d+\.\d+>\]
+    Ancestors: \[#PID<\d+\.\d+\.\d+>\](?:
+    Message Queue Length: 1(?#TODO: Require once depend on 20)|)
```

Comment:
@whatyouhide check out this fancy regex stuff, I look forward to beating you in the next Elixir Cards tournament :1st_place_medal: